cli/lsp/tsc.rs

```diff
@@ -3810,6 +3810,13 @@ fn op_load(
   let state = state.borrow_mut::<State>();
   let mark = state.performance.mark("op_load", Some(&args));
   let specifier = state.specifier_map.normalize(args.specifier)?;
+  if specifier.as_str() == "internal:///missing_dependency.d.ts" {
```

Comment:
I think we should phase out this "missing_dependency.d.ts" thing in the future and instead display errors on the specifiers. It hides valid typescript errors. Changing it would probably be a bit disruptive though.